sdk/ml/azure-ai-ml/azure/ai/ml/_utils/_registry_utils.py

```diff
@@ -199,15 +199,17 @@ def get_storage_details_for_registry_assets(
     )
 
 
-def get_registry_client(credential, registry_name, workspace_location, **kwargs):
+def get_registry_client(credential, registry_name, workspace_location=None, **kwargs):
```

Comment:
```suggestion
def get_registry_client(credential, registry_name, *, workspace_location: Optional[str]=None, **kwargs):
```
Please make it keyword only